### PR TITLE
added alt text 

### PR DESCRIPTION
**Pull request by brynnelliott**
I added alt text for https://swcarpentry.github.io/r-novice-inflammation//fig/rmd-12-supp-factors-updating-factors-1.png

### CODE REVIEW

File: _episodes_rmd/12-supp-factors.Rmd [hunk 1]
```diff
@@ -193,7 +193,7 @@ dat$Gender[dat$Gender == 'M'] <- 'm'
 
 > ## Updating Factors
 >
-> ```{r updating-factors}
+> ```{r updating-factors, fig. alt = "Barplot graph showing blood pressure in males and females.}
```
Comment (brynnelliott) [inline _episodes_rmd/12-supp-factors.Rmd:196]:
```suggestion
> ```{r updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females.}
```

File: _episodes_rmd/12-supp-factors.Rmd [hunk 2]
```diff
@@ -193,7 +193,7 @@ dat$Gender[dat$Gender == 'M'] <- 'm'
 
 > ## Updating Factors
 >
-> ```{r updating-factors}
+> ```{r updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females.}
```
Comment (zkamvar) [inline _episodes_rmd/12-supp-factors.Rmd:196]: From this error:

```
processing file: _episodes_rmd/12-supp-factors.Rmd
(*) NOTE: I saw chunk options "updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females."
 please go to https://yihui.org/knitr/options
 (it is likely that you forgot to quote "character" options)
Error in parse(text = code, keep.source = FALSE) : 
  <text>:1:38: unexpected INCOMPLETE_STRING
1: alist( 'updating-factors', fig.alt = "Barplot graph showing blood pressure in males and females. )
```

```suggestion
> ```{r updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females."}
```

**Comment by zkamvar** [comment]
Note that this will continue to fail even after you fix it due to #566

**Comment by zkamvar** [comment]
Thank you!

**Pull request merged**